test/jdk/sun/management/jmxremote/startstop/JMXStartStopTest.java

```diff
@@ -347,6 +347,8 @@ public synchronized void start() throws InterruptedException, IOException, Timeo
                                                 "the requested port not being available");
                     }
                     pid = p.pid();
+                    jcmd = new ManagementAgentJcmd(p, verbose);
+
```

Comment:
I don't think you want a blank line here.

Reply:
I don't mind the additional space, but ok have removed.